tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/IastInstrumentationUnitTests.cs

```diff
@@ -181,6 +182,8 @@ public void TestInstrumentedUnitTests()
             }
 #endif
             SetEnvironmentVariable("DD_TRACE_LOG_DIRECTORY", Path.Combine(EnvironmentHelper.LogDirectory, "InstrumentedTests"));
+            SetEnvironmentVariable("DD_IAST_DEDUPLICATION_ENABLED", "0");
+            SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "1");
```

Comment:
are we sure we want this? It will slow the tests.

Reply:
I thought that it would be nice for debugging if there was an error, these tests are pretty fast anyway, but I can disable it.